### PR TITLE
Recurrent jobs and improvements 

### PR DESCRIPTION
**Pull request by michelanjos**
New features:

- Be able to schedule a recurrent job
- Be able to remove a recurrent job
- Adding the extension AddMiruTestFixture

### CODE REVIEW

File: src/Miru/Queuing/MiruCron.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        /// <summary>
+        /// Returns cron expression that fires every year at the specified month, day,
+        /// hour and minute in UTC.
+        /// </summary>
+        /// <param name="month">The month in which the schedule will be activated (1-12).</param>
+        /// <param name="day">The day of month in which the schedule will be activated (1-31).</param>
+        /// <param name="hour">The hour in which the schedule will be activated (0-23).</param>
+        /// <param name="minute">The minute in which the schedule will be activated (0-59).</param>
+        public static string Yearly(int month, int day, int hour, int minute) =>
+            $"{(object) minute} {(object) hour} {(object) day} {(object) month} *";
```
Comment (joaofx) [inline src/Miru/Queuing/MiruCron.cs:151]: Maybe it doesn't need to convert to object? I think `$"{minute} {hour} {day} {month} *"` might work. Would you change in other places as well?

File: src/Miru/Queuing/Jobs.cs [hunk 2]
```diff
@@ -6,15 +7,30 @@ namespace Miru.Queuing
...
         public void PerformLater<TJob>(TJob job) where TJob : IJob
         {
             _backgroundJobClient.Enqueue<JobFor<TJob>>(m => m.Execute(job, CancellationToken.None));
         }
+
+        public string PerformLater<TJob>(TJob job, string cronExpression) where TJob : IJob
+        {
+            var jobId = Guid.NewGuid().ToString();
+            _recurringJobManager.AddOrUpdate<JobFor<TJob>>(jobId,m => m.Execute(job, CancellationToken.None),
+                cronExpression, TimeZoneInfo.Local);
```
Comment (joaofx) [inline src/Miru/Queuing/Jobs.cs:27]: When a call to a method is long, would be nice to split one parameter per line:

```csharp
Method(
   p1,
   p2,
   p3);

**Review by joaofx — CHANGES_REQUESTED** [review]
Looks great! Just these minors observations I added but looks great!

Thank you, Michel!

File: src/Miru/Queuing/Jobs.cs (hunk 2 above)
Comment (michelanjos) [reply src/Miru/Queuing/Jobs.cs:27]: I agree with you! For sure, that way will be easier to read and understand.

**Pull request merged**